### PR TITLE
Add helm v3

### PR DESCRIPTION
**Pull request by galal-hussein**

### CODE REVIEW

File: entry [hunk 1]
```diff
@@ -1,5 +1,81 @@
...
+
+helm_repo_init() {
+  # if the chart is url skip repo update
+  if grep -q -e "https\?://" <<< "$CHART"; then
+    echo "chart path is a url, skipping repo update"
+    return
+  fi
+
+  if [ "$HELM" == "helm_v3" ]; then
+    $HELM repo add stable https://kubernetes-charts.storage.googleapis.com/
```
Comment (ibuildthecloud) [inline entry]: The stable repo is effectively deprecated with v3.  Can we check that $REPO starts with stable/ and then only add the stable repo?  For helm 2, keep the existing logic, this is just for helm 3.

File: entry [hunk 2]
```diff
@@ -10,53 +86,15 @@ update-ca-certificates
 tiller --listen=127.0.0.1:44134 --storage=secret &
 export HELM_HOST=127.0.0.1:44134
 
-helm init --skip-refresh --client-only
-helm repo update --strict || helm repo remove stable
-if [ -n "$REPO" ]; then
-    helm repo add ${NAME%%/*} $REPO
+helm_v2 init --skip-refresh --client-only
+EXIST=$(helm_v2 ls --all "^$NAME\$" --output json | jq -r '.Releases | length')
```
Comment (ibuildthecloud) [inline entry]: How much overhead or time does it take to start tiller each time and run `helm ls`?  Does it do anything like create a configmap in k8s?

File: entry (hunk 1 above)
Comment (galal-hussein) [reply entry]: sure will do

File: entry (hunk 2 above)
Comment (galal-hussein) [reply entry]: its really quick process, it starts a daemon that connects to kubernetes to read releases from secrets: 
tiller --listen=127.0.0.1:44134 --storage=secret

and helm init afterwards just initialize the .helm directory within the pod which also a quick process

**Comment by ibuildthecloud** [comment]
LGTM

**Pull request merged**